citrination_api_examples/clients_sequence/1_data_client_api_tutorial.ipynb

```diff
@@ -353,10 +379,28 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dataset_version = data_client.create_dataset_version(dataset_id)\n",
+    "dataset_version = data_client.create_dataset_version(dataset_id=dataset_id)\n",
     "print('Dataset {} is now version {}.'.format(dataset_id, dataset_version.number))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Delete a dataset\n",
+    "\n",
+    "Finally, if you wish to delete a dataset that you own, you can use the `delete_dataset()` method of the `DataClient`. As this is a permanent deletion, please handle with care!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# data_client.delete_dataset(dataset_id=dataset_id)"
```

Comment:
Added a block for `client.data.delete_dataset()`. It's commented out so someone doesn't accidentally do this.